app/views.py

```diff
@@ -25,6 +27,9 @@ def get_user():
 class CopyrightApplicationModelView(ModelView):
     datamodel = SQLAInterface(CopyrightApplication)
     list_columns = LIST_APPLICATION_STATUS
+    validators_columns = {
+        "year_completed": [NumberRange(min=(date_today.year-125), max=date_today.year)]
```

Comment:
Would be cool to pull this out into a validation module and wrap a unit test around it